src/harness/sas.py

```diff
@@ -332,3 +342,113 @@ def _GetDefaultAdminSSLCertPath(self):
   def _GetDefaultAdminSSLKeyPath(self):
     return os.path.join('certs', 'admin_client.key')
 
+
+class HttpServer(httpserver_interface.HttpServerInterface):
+  """Test Harness acting as a Http Server to receive Pull/GET
+   Requests from SAS Under Test
+   parameters: is the dict variable for different types of dump objects.
+   schema of parameters: { "full_activity" : dict to store full activity dump objects,
+   "cbsd_activity": dict to store cbsd activity dump objects,
+   "PPA_activity" : dict to store ppa activity dump objects,
+   "esc_activity" : dict to store esc activity dump objects
+   }
+   schema of full_activity {full activity dump object}
+   schema of cbsd_activity {"id": {cbsd activity dump object}}
+   schema of PPA_activity {"id": {ppa activity dump object}}
+   schema of esc_activity {"id": {esc activity dump object}}
+
+  """
+  def __init__(self, server_details):
+    self._server_details = server_details
+    self.parameters = {}
+
+
+  def _handleRequest(self, getSetupParameters):
+    return lambda *args: HttpServerHandler(getSetupParameters, *args)
```

Comment:
It seems like normally HTTPServer takes as an arg the class which is supposed to handle requests (see the sample code at https://docs.python.org/2/library/basehttpserver.html). Why not use that same style here?